### PR TITLE
Added comments re: package.json

### PR DESCRIPTION
**Pull request by griffinbaker12**
Per my conversation with @adeebshihadeh in Discord, I added comments regarding what each package in the package.json does. I added the comments to a separate MD file b/c the pre-commit hook complained.

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Let's do this? I worry a separate file won't get updated.

https://github.com/c-hive/guides/blob/8dd6863feb69208c2d49fc8d5356cf8e775bde1d/js/best-practices.md#comment-dependencies-in-the-packagejson

File: package.comments.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# package.json comments
+
+## devDependencies
+
+- @solidjs/testing-library: test solid components (only used for App.jsx component)
+- @stylistic/eslint-plugin: enforce stylistic rules beyond standard ESLint
+- @testing-library/jest-dom: testing lib (we don't use)
+- @testing-library/user-event: testing lib (we don't use)
```
Comment (adeebshihadeh) [inline package.comments.md:8]: let's remove anything we don't use!

**Comment by griffinbaker12** [comment]
that's a great reference

**Comment by adeebshihadeh** [comment]
Looks like there's a bad merge?

**Comment by griffinbaker12** [comment]
yeah hmm happened on my other PR too #85

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ jobs:
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           apiToken: ${{ secrets.CLOUDFLARE_PAGES_TOKEN }}
           command: pages deploy dist --project-name=connect --branch=new-connect --commit-dirty=true
-
```
Comment (adeebshihadeh) [inline .github/workflows/build.yaml:53]: revert the changes to this file

**Review by adeebshihadeh — APPROVED** [review]
Looks good, just one minor comment

File: .github/workflows/build.yaml (hunk 2 above)
Comment (griffinbaker12) [reply .github/workflows/build.yaml]: ok cool
